lib/helper/Puppeteer.js

```diff
@@ -2147,7 +2147,7 @@ async function proceedSee(assertType, text, context, strict = false) {
     const locator = new Locator(context, 'css');
     description = `element ${locator.toString()}`;
     const els = await this._locate(locator);
-    assertElementExists(els, 'context element');
+    assertElementExists(els, locator.output);
```

Comment:
I suggest you use a `locator.toString ()`

Because in the locator.js the method `toString` converts the locator via output:
https://github.com/Codeception/CodeceptJS/blob/master/lib/locator.js#L76